source/core/owt_base/LiveStreamIn.cpp

```diff
@@ -741,8 +741,6 @@ bool LiveStreamIn::connect()
 
     m_AsyncEvent << "}";
 
-    av_read_play(m_context);
```

Comment:
I would suggest if m_context->iformat->read_play is nullptr, you still cause this function to avoid impact to non-RTSP stream, as your test might not cover non-RTSP case.

Reply:
OK